interfaces/prompting/errors/errors.go

```diff
@@ -77,20 +79,28 @@ func (e *UnsupportedValueError) Is(target error) bool {
 	return target == ErrUnsupportedValue
 }
 
+func formatUnsupported(unsupported []string) string {
```

Comment:
is this different from strutil.Quoted ?

Reply:
Oh nice, didn't know that already existed. I'll use that. Thanks!